### DISCUSSION TITLE
Create an extra tag for GitHub Actions

### DISCUSSION THREAD

**Discussion by privatenumber** [semantic-release/semantic-release#2068] [Q&A]
## New feature motivation

Seems the convention for creating custom GitHub Actions to be published on the market place is to tag the major version (eg. if the latest version was v1.3.2, it would also be tagged as v1).

## New feature description

For semantic release git plugin to support creating multiple tags. Specifically, force tagging and pushing the second tag.

## New feature implementation

The tricky part is updating the `v1` tag to point at a new commit. I think the best way is to force tag and force push it.

This way, the `v1` tag only needs to be manually added to the GitHub marketplace once and can be updated automatically.

I think if you delete it first and add a new `v1` tag, it unpublishes from the marketplace.

**Comment by gr2m** [comment]
It would not be a semantic-release core feature, but a plugin.

You can also use a branch instead of a tag, which is what I do in my own GitHub Actions. E.g. see an example at https://github.com/octokit/request-action/blob/e8ec792b828bab82a4d436f17f6faacfe343b3ca/.github/workflows/release.yml#L18-L23

I need to manually update the release branch when a breaking change occurs, but that's small overhead and it works great.
